src/web/event-notifications/event-notification-types/SlackNotificationForm.jsx

```diff
@@ -141,9 +174,31 @@ class SlackNotificationForm extends React.Component {
                  label="Custom Message (optional)"
                  type="textarea"
                  bsStyle={validation.errors.custom_message ? 'error' : null}
-                 help={get(validation, 'errors.custom_message[0]', 'Custom message to be appended below the alert title. See https://docs.graylog.org/en/latest/pages/alerts.html#data-available-to-notifications for more details.')}
+                 help={get(validation, 'errors.custom_message[0]', element)}
                  value={config.custom_message || ''}
                  onChange={this.handleChange} />
+
+          <FormGroup>
+            <ControlLabel>Backlog Message Items (optional)</ControlLabel>
+            <InputGroup>
+              <InputGroup.Addon>
+                <input id="toggle_backlog_size"
+                       type="checkbox"
+                       checked={isBacklogSizeEnabled}
+                       onChange={this.toggleBacklogSize} />
+              </InputGroup.Addon>
+              <FormControl type="number"
+                           id="backlog_size"
+                           name="backlog_size"
+                           onChange={this.handleBacklogSizeChange}
+                           value={backlogSize}
+                           min="0"
+                           max="50"
```

Comment:
Don't think we need a max value.

Reply:
its being more defensive, just so we don't accidentally flood the slack client . Let me know your thoughts.

Reply:
My thoughts have been included above.